Formula/gitlab-ci-multi-runner.rb

```diff
@@ -4,8 +4,8 @@ class GitlabCiMultiRunner < Formula
   desc "The official GitLab CI runner written in Go"
   homepage "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner"
   url "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner.git",
-    :tag => "v1.5.2",
-    :revision => "76fdacdf55c5a83249a4bdc5bd8c650e7a5ff796"
+    :tag => "v1.5.3",
+    :revision => "fb49c4781e5040539d3faec87a2a63675d2c9f00"
```

Comment:
Placing anything at the end of a long line (that is long solely due to a long verbatim string) is probably a bad idea. Who would read that.